modules/publication/jsx/viewProject.js

```diff
@@ -258,9 +242,10 @@ class ViewProject extends React.Component {
           text={this.state.formData.leadInvestigator}
         />
         <StaticElement
-         name="leadInvestigatorEmail"
-         label="Lead Investigator Email"
-         text={this.state.formData.leadInvestigatorEmail}
+          id='leadInvestigatorEmail'
```

Comment:
i think you can get ride of this id too

Reply:
thanks @zaliqarosli it should be fixed now.